chat-sdk-ui/src/main/java/co/chatsdk/ui/main/MainActivity.java

```diff
@@ -190,7 +200,7 @@ protected void initViews() {
             public void onTabSelected(TabLayout.Tab tab) {
                 viewPager.setCurrentItem(tab.getPosition());
 
-                InterfaceManager.shared().a. setShowLocalNotifications(showLocalNotificationsForTab(tab));
+                setShowLocalNotificationsForTab(tab);
 
```

Comment:
I create a new function for this because it would make it easier for subclassing. I had to override the onTabSelected method but I still need the code (which I moved to the new function) inside of it, that's why I thought it would make sense to create a new function for it